.github/workflows/integration-quota-tests.yml

```diff
@@ -0,0 +1,106 @@
+name: Integration Quota Tests
```

Comment:
I’m not sure we need a new integration test for this. In my opinion, all relevant cases are already well covered by unit tests, so adding an integration test wouldn’t provide much additional value. Thanks!

Reply:
I dropped the integration test, thanks!